v8/pretty_breadcrumbs/pretty_breadcrumbs.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © 2018 Martin Michlmayr
+
+# Permission is hereby granted, free of charge, to any
+# person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the
+# Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the
+# Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice
+# shall be included in all copies or substantial portions of
+# the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY
+# KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
+# PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS
+# OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+# OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import blinker
+import os.path
+
+from nikola.plugin_categories import SignalHandler
+from nikola import utils
+
+
+class PrettyBreadcrumbs(SignalHandler):
+
+    def _set_pretty_crumbs(self, site):
+        old_get_crumbs = utils.get_crumbs
+
+        if 'PRETTY_BREADCRUMBS_TAG' in self.site.config:
+            tag = self.site.config['PRETTY_BREADCRUMBS_TAG']
+        else:
+            tag = 'crumb'
+
+        def pretty_get_crumbs(path, is_file=False, index_folder=None, lang=None):
+            lang = lang if lang else self.site.default_lang
```

Comment:
use LocaleBorg’s current language in the default case (tons of examples in Nikola core)

Reply:
Ok